src/jira/client/jira-client.ts

```diff
@@ -327,6 +327,19 @@ export const getJiraClient = async (
 				logger.info("Sending remoteLinks payload to jira.");
 				await instance.post("/rest/remotelinks/1.0/bulk", payload);
 			}
+		},
+		permissions: {
+			checkAdmin: async (accountId: string) => {
+				const payload = {
+					accountId,
+					globalPermissions: [
+						"ADMINISTER"
+					]
+				};
+				logger.info("Requesting user permissions from jira.");
+				const response: AxiosResponse = await instance.post("/rest/api/latest/permissions/check", payload);
```

Comment:
It does't look like we need installationId for this call, right? If we don't need it, I suggest to put this code into models/jira-client instead... wdyt?